uaclient/defaults.py

```diff
@@ -6,14 +6,17 @@
 """
 
 UAC_ETC_PATH = "/etc/ubuntu-advantage/"
+UAC_VAR_PATH = "/var/lib/ubuntu-advantage/"
 DEFAULT_CONFIG_FILE = UAC_ETC_PATH + "uaclient.conf"
 DEFAULT_HELP_FILE = UAC_ETC_PATH + "help_data.yaml"
 DEFAULT_UPGRADE_CONTRACT_FLAG_FILE = UAC_ETC_PATH + "request-update-contract"
+MACHINE_TOKEN_FILE = UAC_VAR_PATH + "private/machine-token.json"
```

Comment:
I don't think we can/should hard-code these paths. Probably don't need the UAC_VAR_PATH either because we need to rely on the dynamic/overriding UAConfig.data_dir/data_paths methods to get to both machine-token.json and the new machine-token.json.save files